content/environment_variables.tex

```diff
@@ -31,13 +31,25 @@
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
     & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Controls the size of the symmetric heap per PE. The value
+      set in this environment variable is interpreted as number
+      of bytes, unless the number is followed by a character that
+      acts as a multiplier, where:
+      \begin{itemize}
+        \item k or K multiplies by 2**10 (kilobytes)
+        \item m or M multiplies by 2**20 (megabytes)
+        \item g or G multiplies by 2**30 (gigabytes)
+      \end{itemize}
+      For example, the string 20m returns the integer value 20*2**20,
+      or 20 megabytes. Only one multiplier is recognized, so 20kk
```

Comment:
Instead of "Only one multiplier is recognized, ...", I suggest:
> If more than one multiplier character or any other non-numeric character is provided in this environment variable, the behavior is undefined.

Reply:
changed as  
>Only one multiplier is recognized and any characters following the multiplier are ignored

Reply:
recommended changes are outdated - updated with new texts - marking as resolved. please open new review if needed.